packages/addons/service/minisatip/source/bin/minisatip.start

```diff
@@ -32,7 +32,7 @@ fi
 
 # debug log
 if [ "$MINISATIP_DEBUG" = "true" ]; then
-  MINISATIP_ARG="-l -l -l"
+  MINISATIP_ARG="-l http"
```

Comment:
wasn't aware that this is possible now, I add some followup pr that adds the ability to select the debug "modules" at the gui like we done it at Tvheadend

Reply:
Sounds good.

Reply:
Sounds good.